backend/services/database_service.py

```
@@ -177,6 +178,45 @@ def get_performance_metrics(self, metric_name: str = None, limit: int = 100) ->
+            n_metrics = (
+                session.query(PerformanceMetric)
+                .filter(PerformanceMetric.timestamp < cutoff)
+                .delete(synchronize_session=False)
+            )
+            session.commit()
+            return {
+                "security_findings_deleted": n_findings,
+                "performance_metrics_deleted": n_metrics,
+            }
+        except Exception as e:
+            logger.error(f"Error during cleanup_old_records: {str(e)}")
```

Ruff (0.15.10)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)